ui/drivers/ui_cocoatouch.m

```diff
@@ -448,6 +448,28 @@ - (void)applicationDidBecomeActive:(UIApplication *)application
       [self showGameView];
 }
 
+-(BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey, id> *)options {
+   NSFileManager *manager = [NSFileManager defaultManager];
+   NSError *error;
+   NSString *romPath = [NSHomeDirectory() stringByAppendingPathComponent:@"Documents/RetroArch/roms"];
```

Comment:
Please use `self.documentsDirectory`instead of using the home directory, to properly support tvOS. I don't think we should assume this roms file path too - can you change it to just use  `self.documentsDirectory` like the deprecated method below does?

Reply:
Thank you for the feedback I will change that